src/cards/card.handlers.ts

```diff
@@ -27,11 +59,38 @@ export const updateOne = async (
     const result = await Card.findByIdAndUpdate(req.params.id, req.body, {
       new: true,
     });
-    console.log(result);
+
     if (!result) {
       res.status(404).end();
     } else {
-      res.status(201).json(result);
+      res.status(200).json(result);
+    }
+  } catch (error) {
+    next(error);
+  }
+};
+
+export const deleteOne = async (
+  req: Request<ParamsWithId, {}, {}>,
+  res: Response<{}>,
+  next: NextFunction
+) => {
+  try {
+    const card = await Card.findById(req.params.id);
+    if (!card) {
+      res.status(404).end();
+    } else {
+      if (card.createdAt) {
```

Comment:
**nitpick:** What if we'd like to add more delete validations? We will have ifology here - let's move this validating logic to some separate functions, like `card.validators`